README.md

```diff
@@ -1,3 +1,7 @@
+PLEASE REBASE AND DONT USE MERGE
```

Comment:
What's this?

Reply:
I think back then, some old commits might have some secret that's been scrubbed. This was a reminder for people not to force push to `master` to bring back those commits.

I've already disabled force push & normal push on `master`, this shouldn't be a problem anymore.